config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+      support: "Obwohl RubyGems.org nicht von einem bestimmten Unternehmen ausgeführt wird, haben schon viele uns bisher geholfen. Das aktuelle Design, die Illustrationen und die Front-End-Entwicklung für dieser Webseite wurden von %{dockyard} erstellt. %{github} hat auch von unschätzbarem Wert uns geholfen den Code mit anderen Entwicklern zuteilen, um an dem Code leichter zusammenarbeiten. Die Website wurde mit %{heroku} gestartet, dessen großen Dienst uns geholfen, RubyGems als eine praktikable Lösung zu erweisen, wo die ganze Gemeinschaft darauf verlassen kann."
+      technical: "Einige Einblicke in die technischen Aspekte dieser Webseite: Es ist 100% in Ruby geschrieben. Der Hauptseite ist verwendet die %{rails}-Software. RubyGem nutzt %{sinatra}, um die Gems bereitzustellen. Die Gems werden in %{s3} gehostet und die Zeit zwischen der Veröffentlichung eines neuen Gems und nachdem er bereit zur Installation freigegeben ist, ist minimal. Für mehr Informationen siehe %{source_code} (%{license}) auf GitHub."
+
+  passwords:
+    edit:
+      title: Zurücksetzen des Passworts
+      submit: Speichern dieses Passworts
+    new:
+      title: Ändere dein Passwort
+      will_email_notice: Wir senden dir einen Link damit du dein Passwort ändern kannst.
+      submit: Zurücksetzen des Passworts
+
+  profiles:
+    edit:
+      title: Bearbeite Profil
+      hide_email: Verberge E-Mail in öffentliches Profil
+      reset_password:
+        title: Zurücksetzen des Passworts
+        text: "Brauchst du ein neues Passwort? Kein Problem. %{link}"
+        link_text: Jetzt ein neues Passwort zuschicken lassen.
+      api_access:
+        title: API-Zugang
+        key_is: "Dein API-Schlüssel ist %{key}."
+        credentials: "Wenn du %{gem_commands_link} über das Terminal oder über das CMD verwenden möchtest, dann brauchst du die %{gem_credentials_file}-Datei, die du mit dem folgenden Befehl erzeugen kannst:"
+        link_text: Gem-Befehle
+        reset: Zurücksetzen meines API-Schlüssels
+        confirm_reset: Bist du sicher? Das kann nicht rückgängig gemacht werden.
+
+  rubygems:
+    dependencies:
+      header: "%{title} Abhängigkeiten"
+    edit:
+      title: "Bearbeite Links"
+      subtitle: "für %{gem}"
+      other_fields_notice: Alle anderen Felder können nur durch eine gepushte, aktualisierte Version deines Gems bearbeitet werden.
+    index:
+      title: 'Gems'
+      downloads: Downloads
+    show:
+      not_hosted_notice: Dieses Gem wird nicht aktuell auf RubyGems.org gehostet.
+      install: installieren
+#      yanked_notice: "This gem has been yanked, and it is not available for download directly or for other gems that may have depended on it."
+      authors_header: Autoren
+      downloads_for_this_version: "Für diese Version"
+      owners_header: Besitzern
```

Comment:
Depends on the context of where this is inserted, but this should probably say "Besitzer" (no trailing "n")